api/spec/requests/spree/api/stores_controller_spec.rb

```diff
@@ -76,7 +79,7 @@ module Spree
           code: "spree123",
           name: "Hack0rz",
           url: "spree123.example.com",
-          mail_from_address: "me@example.com"
+          mail_from_address: "me@example.com",
```

Comment:
the comma without any subsequent line makes me think that maybe you meant to add the `bcc_email` field here?

Reply:
Good catch! I ultimately decided it was unnecessary to add the field there (since bcc_email isn't required to create a store, which is what it's testing), but forgot to remove the comma. 

As for the BCC email not being capitalized, I uploaded an outdated screenshot - serves me right for not checking first. It's properly capitalized in this version. I'll update my PR with the latest screenshot.